keyboards/gboards/gergoplex/rules.mk

```
@@ -0,0 +1,18 @@
+MCU = atmega32u4
+
+BOOTLOADER = atmel-dfu
+
+CUSTOM_MATRIX    = lite
+MOUSEKEY_ENABLE  = yes  # Mouse keys
+COMBO_ENABLE     = yes
+EXTRAKEY_ENABLE  = yes
+CONSOLE_ENABLE   = no
+NKRO_ENABLE      = yes  # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+COMMAND_ENABLE   = yes
+BOOTMAGIC_ENABLE = lite
+
+LAYOUTS = split_3x5_3
+
+DEBOUNCE_TYPE  = eager_pr
```

```suggestion
DEBOUNCE_TYPE  = sym_eager_pr
```
This fails building with current `develop` since the old names seem to be removed.

---

Thanks!